README.md

```diff
@@ -125,7 +125,7 @@ Microsoft Cloud for Financial Services provides capabilities to manage data to d
 
 - Customer and employee experience
   - [Unified customer profile](./fsi/solutions/unifiedCustomerProfile)
-  - [Customer onboarding](./fsi/solutions/customerOnboarding)
+  - [Customer onboarding](./fsi/solutions/customerOnboarding/readme.md)
```

Comment:
```suggestion
  - [Customer onboarding](./fsi/solutions/customerOnboarding)
```

Reply:
@marvinbuss Links wasnt working that way that is why I added the readme.md